README.md

```diff
@@ -9,17 +9,35 @@ Discussions for AI conformance requirements are now tracked using the
 GitHub Project, shifting from the original
 [Google Doc](https://docs.google.com/document/d/1hXoSdh9FEs13Yde8DivCYjjXyxa7j4J8erjZPEGWuzc/edit?tab=t.0).
 
-Each requirement is tracked as a GitHub issue with the following workflow:
-- **Todo**: Requirements currently under discussion.
-- **In Progress**: Requirements that have been accepted by the working group.
-- **Done**: Requirements that have been added to the [cncf/ai-conformance](https://github.com/cncf/ai-conformance) repository.
+Each requirement is tracked as a GitHub issue with the following workflow and status label:
+- **Implementable**: Has been accepted by the working group for a Kubernetes release.
+- **Implemented**: Has been part of one or more Kubernetes releases in the [cncf/ai-conformance](https://github.com/cncf/ai-conformance) repository and has graduated to MUST. Implementation is complete. Further changes should be made via new KARs.
 
-To participate, please comment on the relevant GitHub issues.
+To participate, please comment on the relevant GitHub issues and pull requests.
 Unresolved items will be discussed during the
 [recurring WG meeting](https://github.com/kubernetes/community/tree/master/wg-ai-conformance#meetings).
 
+### Process Details
+
+This process adopts the Kubernetes Enhancement Proposal (KEP) process as the basis for managing the lifecycle of Kubernetes AI conformance Requirements (KARs), including review, discussion, and approval.
+
+- **Requirement Proposal**: Propose each new Kubernetes AI conformance Requirement (KAR) as a GitHub issue, targeting a specific Kubernetes release. Requirements will be tracked and reviewed individually. This is similar to a KEP, but tracked as a KAR with an issue in this wg-ai-conformance repo.
```

Comment:
Should this mention creating a PR for KAR as well? And we use GitHub issues for tracking progress (similar to the KEP process). We're also going to add milestone and SHOULD/MUST status to the tracking issue.